includes/Core/User_Input/User_Input.php

```diff
@@ -126,10 +127,13 @@ public static function get_questions() {
 	 */
 	public function get_answers() {
 		$questions = static::$questions;
-		$settings  = array_merge(
-			$this->site_specific_answers->get(),
-			$this->user_specific_answers->get()
-		);
+		$settings  = $this->site_specific_answers->get();
```

Comment:
@sashadoes why did you change this?

Reply:
This fixes User_InputTest.php tests that check if arrays are equal. 
Which depends on the Feature

Reply:
Then we need to enable this feature flag for those tests. Could you please revert changes in this file and fix broken tests?